apps/wallet-mobile/src/features/Scan/common/useTriggerScanAction.tsx

```diff
@@ -65,16 +62,17 @@ export const useTriggerScanAction = ({insideFeature}: {insideFeature: ScanFeatur
           }
           if ('memo' in scanAction.params) memoChanged(scanAction.params?.memo ?? '')
         }
+
+        navigateTo.startTransfer()
         break
       }
 
       case 'send-only-receiver': {
-        navigateTo.back()
-        navigateTo.send()
-
-        if (insideFeature !== 'send') resetSendState()
+        if (insideFeature !== 'send') resetTransferState()
 
         receiverResolveChanged(scanAction.receiver)
+
+        navigateTo.startTransfer()
```

Comment:
this fix the glitchy issue when going back on QR scan, navigation state was breaking the animation

Reply:
@rahulnr7 this needs testing